cartridges/app_custom2/cartridge/templates/default/form.isml

```diff
@@ -0,0 +1,36 @@
+<iscontent type="text/html" charset="UTF-8" compact="true" />
+
+<isdecorate template="common/layout/page">
+  <div class="container">
+    <div class="card">
+      <form action="${pdict.actionUrl}" class="login" method="POST" name="profileForm">
+
+        <div class="form-group required">
+          <label> First name: </label>
+          <input type="input" id="firstName" class="form-control" name="firstName"
+            value="${pdict.profileForm.customer.firstname.value}" />
+
+          <label> Last name: </label>
+          <input type="input" id="lastName" class="form-control" name="lastName"
+            value="${pdict.profileForm.customer.lastname.value}" />
+
+          <label> Email: </label>
+          <input type="input" id="email" class="form-control" name="email"
+            value="${pdict.profileForm.customer.email.value}" />
+        </div>
+
+        <div class="form-group required">
+          <label> Interests: </label>
+          <input type="input" id="interests" class="form-control" name="interests"
+            value="${pdict.profileForm.training2.interests.value}" />
+
+          <label> Favorite movie: </label>
+          <input type="input" id="favoriteMovie" class="form-control" name="favoriteMovie"
+            value="${pdict.profileForm.training2.favoriteMovie.value}" />
```

Comment:
try not hardcoding the name and value, but use the attributes property
`<input type="input" class="form-control" 
            <isprint value="${pdict.profileForm.training2.favoriteMovie.attributes}" encoding="off" />/>`